cmd/kube-apiserver/app/server.go

```diff
@@ -269,7 +269,6 @@ func CreateKubeAPIServerConfig(s completedServerRunOptions) (
 			StorageFactory:          storageFactory,
 			EventTTL:                s.EventTTL,
 			KubeletClientConfig:     s.KubeletConfig,
-			EnableLogsSupport:       s.EnableLogsHandler,
```

Comment:
revert this change (and all the other changes that plumb and set up handlers based on this value)